turborepo-tests/examples/package.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "turborepo-examples-tests",
```

Match file system layout?

```suggestion
  "name": "turborepo-tests-examples",
```